python/vitables/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vitables
+version             3.0.3
+categories          python science
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3
```

Comment:
this should become `GPL-3+` as you can see from the [`pyproject.toml` file](https://github.com/uvemas/ViTables/blob/319271e844c40163936811c64ee91a671c9a20f0/pyproject.toml#L11)